canonical/juju-dashboard
# WD-5051 - Split application tab sub components

## Done

- Split up the application tab sub-components into their own files.
- Add additional tests.
- Move search results to a file in preparation for the search API.

## QA

- Go to the application tab in a model.
- Search for an app.
- Tick an app and check that clicking "Run actions" opens the panel.

## Details

https://warthogs.atlassian.net/browse/WD-5051
